release-notes/DIS/DIS_3.0.md

```diff
@@ -4,6 +4,12 @@ uid: DIS_3.0
 
 # DIS 3.0
 
+## New features
+
+### Validator
```

Comment:
Just to be sure: is it OK that no RN number is mentioned for this entry?

Reply:
Yeah as we don't do RNs for NuGet updates.